redhat-operators/businessautomation-operator/businessautomation-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,206 @@
+      mediatype: image/svg+xml
+  labels:
+    alm-owner-businessautomation: businessautomation-operator
+    operated-by: businessautomation-operator.v1.0.0
+  selector:
+    matchLabels:
+      alm-owner-businessautomation: businessautomation-operator
+      operated-by: businessautomation-operator.v1.0.0
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: false
+    - type: AllNamespaces
+      supported: true
+  customresourcedefinitions:
+    owned:
+      - version: v1
+        kind: KieApp
+        description: An instance of KieApp
+        displayName: KieApp
+        name: kieapps.app.kiegroup.org
+        resources:
+          - kind: DeploymentConfig
+            version: apps.openshift.io/v1
+          - kind: Secret
+            version: v1
+          - kind: PersistentVolumeClaim
+            version: v1
+          - kind: ServiceAccount
+            version: v1
+          - kind: Role
+            version: authorization.openshift.io/v1
+          - kind: RoleBinding
+            version: authorization.openshift.io/v1
+          - kind: Service
+            version: v1
+          - kind: Route
+            version: route.openshift.io/v1
+          - kind: BuildConfig
+            version: build.openshift.io/v1
+          - kind: ImageStream
+            version: image.openshift.io/v1
+        specDescriptors:
+          - description: Environment of PAM to deploy.
+            displayName: Environment
+            path: environment
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:label"
+          - description: PAM version to be installed.
+            displayName: Version
+            path: commonConfig.version
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:label"
+        statusDescriptors:
+          - description: Deployments for the KieApp environment.
+            displayName: Deployments
+            path: deployments
+            x-descriptors:
+              - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
+          - description: The address for accessing workbench, if it is deployed.
+            displayName: Workbench URL
+            path: consoleHost
+            x-descriptors:
+              - "urn:alm:descriptor:org.w3:link"
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+        - serviceAccountName: business-automation-operator
+          rules:
+            - apiGroups:
+                - ""
+              resources:
+                - "*"
+              verbs:
+                - "*"
```

Comment:
Does this operator need this role? It seems to be broader than the one below which accesses the same apiGroup.

Reply:
i believe that this had more to do w/ us running into this bug and having to elevate perms in order to run the operator.
https://github.com/operator-framework/operator-lifecycle-manager/pull/673

Reply:
i've cleaned this up